src/notify/subscription/db.go

```diff
@@ -14,26 +14,26 @@ var prepStmts struct {
 func CreateDB(db *sql.DB) error {
 	myDB = db
 
-	_, err := myDB.Exec(`
-              CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
-              CREATE TABLE IF NOT EXISTS subscriptions (
-                id UUID PRIMARY KEY,
-                event_type VARCHAR(64),
-                context VARCHAR(64),
-                account_id UUID,
-                created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP )
-              `)
-	if err != nil {
+	if _, err := myDB.Exec(`
+      CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+      CREATE TABLE IF NOT EXISTS subscriptions (
+        id UUID PRIMARY KEY,
+        event_type VARCHAR(64),
+        context VARCHAR(64),
+        account_id UUID,
+        created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP )
+    `); err != nil {
```

Comment:
I think these poor SQL examples would be more palatable if the SQL were externalized.